src/components/ZclCommandManager.vue

```diff
@@ -35,7 +35,10 @@ limitations under the License.
           </q-tr>
         </template>
         <template v-slot:body="props">
-          <q-tr :props="props">
+          <q-tr
+            :props="props"
+            v-show="props.row.source === 'server' || !enableServerOnly"
```

Comment:
Should we just be showing commands which have source as server?
Are there no response commands which should be shown when command source is client?

Reply:
I thought that it was discussed we don't need to show Client commands but not 100 percent sure